package.json

```diff
@@ -57,7 +57,7 @@
     "@types/styled-components": "^5.1.25",
     "ajv": "^8.11.0",
     "concurrently": "^7.2.2",
-    "electron": "^14.2.9",
+    "electron": "^20.0.2",
```

Comment:
wow thats a huge bump. Is everything working properly? The sandboxing and context isolation comes to mind tha always breaks with major version bumps.

Reply:
I've double-checked, and manual testing and the e2e tests all seem to be ok.